### ISSUE TITLE
Layout issues

### ISSUE THREAD

**Issue by Genhain** [QULab/QULQuestionnaire-iOS#2]
![screen shot 2017-01-19 at 5 48 12 pm](https://cloud.githubusercontent.com/assets/2082208/22101712/8a76b810-de6f-11e6-865e-119133035adc.png)

As you can see, I tried using this in my project and i got the above, The top bar is all cluttered, The check boxes are invisible and the text does not autosize to accommodate.

So then i tried the example to see what i was doing wrong and it looks exactly the same. What am i doing or not doing that is causing this?

**Comment by lawmaestro** [comment]
Hey. I'm also seeing this same behavior :(

**Comment by Genhain** [comment]
@lawmaestro Yeah, our team made a fork for our own use try this link https://github.com/xelven/QULQuestionnaire-iOS
